Src/BootCamp.Chapter/Class3.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Class3
```

Comment:
Name this class "Lesson3"

Reply:
done, is changed. I did not read the homework properly. Thanks for the remark